cura/BuildVolume.py

```diff
@@ -189,6 +192,11 @@ def _onSettingPropertyChanged(self, setting_key, property_name):
         if setting_key == "print_sequence":
             if Application.getInstance().getGlobalContainerStack().getProperty("print_sequence", "value") == "one_at_a_time":
                 self._height = self._active_container_stack.getProperty("gantry_height", "value")
+                Message(catalog.i18nc(
+                    "@info:status",
+                    "The build volume height has been reduced due to the value of the"
+                    " \"Print Sequence\" setting to prevent the gantry from colliding"
+                    " with printed objects."), lifetime=10).show()
```

Comment:
The 30 seconds was introduced because people mentioned that the 10 seconds was too short, because they often missed important messages. 
I don't really mind which of the two we go for, but it should be a concious decision. 
@fieldOfView What's your opinion on this?